docker-compose/obkm/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2019 WSO2, Inc. (http://wso2.com)
```

Comment:
@Akila94 shall we please update the license header year to latest?

Update all applicable locations in the PR.

Reply:
Changed, thanks.